service/api_patch.go

```diff
@@ -147,21 +147,22 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var moduleName, patchContent, githash string
+	var moduleName, patchContent, githash, message string
 
 	if r.Header.Get("Content-Type") == formMimeType {
-		moduleName, patchContent, githash = r.FormValue("module"), r.FormValue("patch"), r.FormValue("githash")
+		moduleName, patchContent, githash, message = r.FormValue("module"), r.FormValue("patch"), r.FormValue("githash"), r.FormValue("message")
```

Comment:
nit: can you put this on multiple lines?

Reply:
done.